modules/node_modules/@frogpond/ccc-rss-feed/index.js

```diff
@@ -42,13 +42,23 @@ export function convertRssItemToStory(item) {
 		: content.substr(0, 250)
 	excerpt = JSDOM.fragment(excerpt).textContent.trim()
 
+	let featuredImage = null
+	if (item.querySelector('enclosure')) {
+		let featuredMediaInfo = item.querySelector('enclosure')
+		let containsImage = featuredMediaInfo.getAttribute('type').startsWith('image')
```

Comment:
What about `startsWith('image/')`? I don't quite know how strict mime types usually are, but I think PNG is `image/png`, and I'd like to avoid things like `imagent/foo` being caught here…

Reply:
That's a great suggestion!